docs/ReleaseHistory.md

```diff
@@ -11,6 +11,12 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- BRK: Regular expression syntax has been standardized in JSON to conform to how the overwhelming majority of patterns were already defined.
+  - `refine` is used now used throughout as the name of the capture group used to isolate an actual find from the full expression that also matches delimiting characters. `secret` was previously used in some instances.
```

Comment:
>refine [](http://example.com/codeflow?start=5&length=6)

I hate to mention this but citing the specific rules that were subject to the changes you've made would be good documentation to have here.

Reply:
Done.